packages/gradient-block/src/settings.ts

```diff
@@ -0,0 +1,123 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import {
+    Choice,
+    SwitchSize,
+    appendUnit,
+    defineSettings,
+    maximumNumericalOrPixelOrAutoRule,
+    maximumNumericalRule,
+} from '@frontify/guideline-blocks-settings';
+
+import { IconEnum } from '@frontify/fondue';
+import { GradientHeight, GradientOrientation } from './types';
+import { DEFAULT_HEIGHT_VALUE, DEFAULT_ORIENTATION_VALUE } from './constants';
+
+export const IS_CUSTOM_HEIGHT_ID = 'isHeightCustom';
+export const HEIGHT_CUSTOM_ID = 'heightCustom';
+export const HEIGHT_SIMPLE_ID = 'heightSimple';
+
+export const IS_CUSTOM_ORIENTATION_ID = 'isOrientationCustom';
+export const ORIENTATION_CUSTOM_ID = 'orientationCustom';
+export const ORIENTATION_SIMPLE_ID = 'orientationSimple';
+
+export const settings = defineSettings({
+    basics: [
+        {
+            id: 'gradientInspect',
+            type: 'sectionHeading',
+            label: 'Inspect',
+            blocks: [
+                {
+                    id: 'displayCss',
+                    label: 'Display CSS Code',
+                    type: 'switch',
+                    size: SwitchSize.Small,
+                    defaultValue: true,
```

Comment:
According to figma the default would be off
```suggestion
                    defaultValue: false,
```

Reply:
adjusted